lib/propagate-steps.js

```diff
@@ -150,7 +162,15 @@ const propagateNamespaces = (instances, instance) => {
  *  Updating of instances is achieved by setting properties directly on uncloned instances
  **/
 const propagateUrls = (instances, instance) => {
-  const serviceRoot = '/'
+  let serviceRoot = ''
+  if (instance.mountPoint) {
+    const mountPointInstance = instances[instance.mountPoint]
+    propagateUrls(instances, mountPointInstance)
+    serviceRoot = instances[instance.mountPoint].url
+  }
+  if (!serviceRoot.endsWith('/')) {
```

Comment:
How come?

Reply:
If a page has no parent and its url isn't absolute, we need to add its url to either the root url or the mountPoint instance's url.

The root url is obviously '/', but the mountInstance will be something like '/mountpoint'

This just ensures there's a slash between the 2 parts